native/packages/react-native-backpack/libraries/Panel/android/gradlew.bat

```diff
@@ -0,0 +1,84 @@
+@if "%DEBUG%" == "" @echo off

```

Comment:
are we still keeping this? we don't support windows for developments, we can safely gitignore this

Reply:
removed